Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1281,6 +1292,51 @@ def timestamp_length_equalization(timestamp1, timestamp2):
     return int(timestamp1), int(timestamp2)
 
 
+def change_host_group(method: str,
+                      host_group_id: str = None,
+                      name: str = None,
+                      group_type: str = None,
+                      description: str = None,
+                      assignment_rule: str = None) -> CommandResults:
+    allowed_methods = {'POST', 'PATCH'}
+    if method not in allowed_methods:
+        raise DemistoException(f'CrowdStrike Falcon error: method should be in {allowed_methods}')
```

Comment:
Please replace the `method` arg with a bool flag. No need to handle an incorrect value.